addons/mail/static/src/discuss/core/public_web/poll.xml

```diff
@@ -0,0 +1,61 @@
+                    <t t-if="state.showResults" t-call="mail.PollAnswer-showingResults"/>
+                    <t t-else="" t-call="mail.PollAnswer-voting"/>
+                </div>
+                <div class="d-flex align-items-center mt-2">
+                    <p class="small text-muted m-0">
+                        <t t-esc="pollVoteCountText"/>
+                        &#8729;
+                        <t t-if="props.poll.end_message_id">Poll closed</t>
+                        <t t-else="" t-esc="state.remainingTimeText"/>
+                    </p>
+                    <span class="flex-grow-1"/>
+                    <t t-if="!props.poll.end_message_id">
+                        <t t-if="!props.poll.selfAlreadyVoted">
+                            <t t-if="state.showResults">
+                                <button class="btn btn-secondary" t-on-click="toggleShowResults">Go back to vote</button>
+                            </t>
+                            <t t-else="">
+                                <a href="#" class="me-2" t-on-click="toggleShowResults">View results</a>
+                                <button class="btn btn-primary" t-att-disabled="voteButtonDisabled" t-on-click="onClickVote">Vote</button>
+                            </t>
+                        </t>
+                        <button t-else="" class="btn btn-secondary" t-on-click="props.poll.removeVote">Remove Vote</button>
+                    </t>
+                </div>
+            </div>
+        </div>
+    </t>
+    <t t-name="mail.PollAnswer-voting">
+        <label class="o-discuss-PollAnswer d-flex justify-content-between p-2 cursor-pointer rounded border" t-att-class="{
+            'border-primary': state.selectedAnswerIds.has(answer.id),
+            'border-secondary': !state.selectedAnswerIds.has(answer.id),
+        }">
+            <span class="fw-bold me-1" t-esc="answer.text"/>
+            <input class="form-check-input" t-att-disabled="state.voting" t-att-checked="state.selectedAnswerIds.has(answer.id)" t-on-change="(ev) => this.onAnswerCheckboxToggle(answer.id, ev.target.checked)" type="checkbox"/>
+        </label>
+    </t>
+    <t t-name="mail.PollAnswer-showingResults">
+        <div class="d-flex justify-content-between p-2 rounded border">
+            <span class="fw-bold me-1" t-esc="answer.text"/>
+            <span class="flex-shrink-0">
+                <span class="fw-bold me-1 smaller" t-esc="answerVoteCountText(answer)"/>
+                <span class="fw-bold">&#8729; <t t-esc="answer.vote_percentage"/> %</span>
+                <i t-if="answer.is_selected_by_self" class="ms-1 fa fa-lg fa-check-circle-o" t-att-class="{
+                    'text-success': answer.eq(props.poll.winning_answer_ids[0]), 
+                    'text-primary': !answer.eq(props.poll.winning_answer_ids[0]),
+                }"/>
+            </span>
```

Comment:
This doesn't work: no `fa-check` visually on self selected answers, it looks like `is_selected_by_self` is unset in code.